pytorch_toolkit/instance_segmentation/segmentoly/utils/onnx.py

```diff
@@ -0,0 +1,78 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+import onnx
+import torch
+import logging
+
+
+# FIXME. Remove this?
```

Comment:
Please, check all `FIXME` and `TODO` comments and fix them if possible.

Reply:
@AlexanderDokuchaev All comments with `FIXME` and `TODO` were fixed or removed.